### ISSUE TITLE
GridView keyboard navigation doesn't work

### ISSUE THREAD

**Issue by akoeplinger** [MahApps/MahApps.Metro#20]
When using the Metro style on a GridView inside a ListView, the keyboard navigation doesn't work anymore. Nothing happens when pressing the up/down arrow keys instead of moving through the list of items.
